particle-node/src/builtins_loader.rs

```diff
@@ -114,6 +115,43 @@ fn get_blueprint_id(modules: &Vec<Module>, name: String) -> Result<String> {
     Ok(hash_dependencies(facade, deps_hashes).to_string())
 }
 
+fn load_scheduled_scripts(path: &PathBuf) -> Result<Vec<ScheduledScript>> {
+    let mut scripts = vec![];
+    if let Some(files) = list_files(&path.join("scheduled")) {
+        for path in files.into_iter() {
+            let data = fs::read_to_string(path.clone())?;
+            let name = path
+                .file_stem()
+                .ok_or(eyre!("invalid path"))?
+                .to_str()
+                .ok_or(eyre!("path to {} contain non-UTF-8 character"))?
```

Comment:
```suggestion
                .ok_or(eyre!("path to {} contains non-UTF-8 character"))?
```

Maybe move all this `file_steam.ok_or().to_str().ok_or()` shenanigans to utils function?